x-pack/plugin/build.gradle

```diff
@@ -143,6 +143,7 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
   task.skipTest("indices.freeze/10_basic/Test index options", "#70192 -- the freeze index API is removed from 8.0")
   task.skipTest("sql/sql/Paging through results", "scrolling through search hit queries no longer produces empty last page in 8.2")
   task.skipTest("service_accounts/10_basic/Test get service accounts", "new service accounts are added")
+  task.skipTest("spatial/70_script_doc_values/diagonal length", "precision changed in 8.4.0")
```

Comment:
Do we permanently skip this test? I see you changed the test result in the yaml file so do we need to skip this at all?

Reply:
We are skipping the V7 compatibility test (yamlRestTestV7CompatTransform) . It seems this mode is running the test on the v7 branch against this cluster. That currently fails because the change in precision which is fine.